allianceauth/services/modules/teamspeak3/manager.py

```diff
@@ -181,16 +182,18 @@ def _sync_ts_group_db(self):
 
     def add_user(self, username):
         username_clean = self.__santatize_username(username[:30])
+        user = User.objects.get(username=username)
```

Comment:
This method is called with a [formatted service name](https://github.com/allianceauth/allianceauth/blob/master/allianceauth/services/modules/teamspeak3/views.py#L25) and not a username, so this won't work.

What I think would make more sense is to pass the `User` object directly from the view to the manager. There's no good reason to pass just a username, just \~legacy code\~